docs/code.rst

```diff
@@ -278,6 +278,7 @@ Operators
 .. autoclass:: airflow.contrib.operators.vertica_to_mysql.VerticaToMySqlTransfer
 .. autoclass:: airflow.contrib.operators.wasb_delete_blob_operator.WasbDeleteBlobOperator
 .. autoclass:: airflow.contrib.operators.winrm_operator.WinRMOperator
+.. autoclass:: airflow.contrib.operators.telegram_operator.TelegramAPIOperator
```

Comment:
Please add the class in alphabetical order. First, according to class names, and then according to the module path.